### DISCUSSION TITLE
More Commands/keyboard shortcuts?

### DISCUSSION THREAD

**Discussion by tinoargentino** [cline/cline#529] [Feature Requests]
3 easy ones that would be very useful as vscode commands that can be binded to keyboard shortcuts would be
1. new task
2. Settings
3. History

Would be great!!!

**Comment by mxro** [comment]
So far I could only get to work to open Cline when the editor is active by mapping the command `"workbench.view.extension.claude-dev-ActivityBar"` - however, that doesn't focus on the text input box if there is already a conversation with Cline ongoing. Would be great to have more options here!

**Comment by dcbartlett** [comment]
https://github.com/cline/cline/pull/1626 <-- This will implement a shortcut to switch Plan/Act modes.
